Updated github user docs (#447)
spatialos/UnrealGDK: README.md

# The SpatialOS Game Development Kit for Unreal (alpha)

> The SpatialOS Game Development Kit (GDK) for Unreal is an Unreal Engine 4 (UE4) [plugin](https://docs.unrealengine.com/en-us/Programming/Plugins) which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API.

Reviewer: [plugin (Unreal documentation)]

The GDK offers:

> * **Multi-server support**: leveraging our cloud platform SpatialOS (SpatialOS documentation), the GDK turns Unreal’s single-authority engine into a distributed one,  enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.

Reviewer 2: ```suggestion
* **Multi-server support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK turns Unreal’s single-authority engine into a distributed one, enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.
```

> * **An Unreal-native experience**: keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features Unreal offers out of the box, including transform synchronization, character movement, client-side prediction.

Reviewer 2: ```suggestion
* **An Unreal-native experience**: keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features Unreal offers out of the box, including transform synchronization, character movement, and client-side prediction.
```

> * **An Unreal-native experience**: keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features Unreal offers out of the box, including transform synchronization, character movement controller, client-side prediction.

Reviewer 3: Technically there is no "character movement controller"

We can just say "character movement". Also can add stuff like ability system maybe, although it doesn't work across boundaries yet.

> * **Multi-server support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK turns Unreal’s single-authority engine into a distributed one,  enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.

Reviewer: one,  enabling
(Double space)

> * **Multi-server support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK turns Unreal’s single-authority engine into a distributed one,  enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.

Reviewer 3: We say "single-authority engine into distributed" twice in this section, let's change this one slightly. 

```leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK enables usage of more than one Unreal game server in a single instance so that games can have more players, Actors, and gameplay systems than previously possible.``

> * **Easy getting started**: we have made sure it’s easy to get started with the GDK by including a Starter Project which you can use as a tour of SpatialOS and a base for your own game, as well as a guide to porting your current multiplayer Unreal game to run on SpatialOS.

Reviewer 3: "Easy to get started?", no? @ElleEss

Reviewer 2: Perhaps "An easy onboarding experience"? Or maybe "An easy experience getting started"?

> >This is an [alpha (SpatialOS website)](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK, pending stability and performance improvements. The API may change as we learn from feedback  - see the guidance on [Recommended use](#recommended-use), below.

Reviewer 2: ```suggestion
>This is an [alpha (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal, pending stability and performance improvements. The API may change as we learn from feedback  - see the guidance on [Recommended use](#recommended-use), below.
```

> >This is an [alpha (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK, pending stability and performance improvements. The API may change as we learn from feedback  - see the guidance on [Recommended use](#recommended-use), below.

Reviewer: SpatialOS Unreal GDK
->
SpatialOS GDK for Unreal

----

> * [Get started](https://docs.improbable.io/unreal/latest/content/get-started/introduction) (on the documentation website)

Reviewer 2: ```suggestion
* [Get started](https://docs.improbable.io/unreal/latest/content/get-started/introduction) (on the SpatialOS documentation website)
```

> * [Documentation](https://docs.improbable.io/unreal/latest) (on the documentation website)

Reviewer 2: ```suggestion
* [Documentation](https://docs.improbable.io/unreal/latest) (on the SpatialOS documentation website)
```

## Give us feedback
We have released the GDK for Unreal this early in development because we want your feedback. Please come and talk to us about the software and the documentation via: [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) - [Forums](https://forums.improbable.io/) - [GitHub issues in this repository](https://github.com/spatialos/UnrealGDK/issues).

> ## Where to get the GDK and example projects

Reviewer 3: Should we say "related projects" instead? Some of these are not really example projects.

## Where to get the GDK and starter projects
The GDK and its starter projects are available on GitHub.
* [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK)
* [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
* [Starter Project](https://github.com/spatialos/UnrealGDKStarterProject)
* [Third-Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)</br>

> **NOTE:** This link may give you a 404. 

Reviewer 4: This section referring to `This link may give you a 404.` is incorrect. This is only for the UnrealEngine fork which is already stated.

> ## Where to get the GDK and starter projects

Reviewer: starter projects .<- Didn't you want to rename this title to something which didn't say "starter project"?

Reviewer: sample projects

> The GDK and its starter projects are available on GitHub.

Reviewer: starter projects -> sample projects

* [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
* [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK)
* [Starter Project](https://github.com/spatialos/UnrealGDKStarterProject)

> * [Third-Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)

Reviewer: Game -> game

* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)</br>

> **NOTE:** This link may give you a 404. 

Reviewer: This note needs to sit under the UE4 link - it's why it was last before - I think Giray re-ordered but the 404 note doesn't make sense now, I have just realised.

Reviewer 3: Yes

The GDK and its starter projects are available on GitHub.
* [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
* [Starter Project](https://github.com/spatialos/UnrealGDKStarterProject)
* [Third-Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)

> * [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK) </br>

Reviewer 2: For consistency with my comments in the other README docs, we could move this up so it's just below the GDK bullet point. (Or, if you didn't make that change in the other docs, leave it here as well.)

* Version: alpha (stability and performance improvements pending)

> * GDK repository: [github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)

Reviewer 4: GDK repository is now listed above so not needed here.

Author: I think we might as well have it here

Reviewer: It's there for form as you usually have your repo URL on your repo readme so you can find it again when you clone this baby.
